src/components/TimelineItemList.svelte

```diff
@@ -129,6 +129,10 @@
       viewAddFilterToRow([event.detail.item], typeName, {}, event.detail.row?.id, event.detail.layer);
     }
   }
+
+  function hasDescription(item: TimelineItemType): item is TimelineItemType & { description: string } {
```

Comment:
It would be more clear if this typeguard returned the actual types that contain a `description`
```
  function hasDescription(item: TimelineItemType): item is ResourceType | ActivityType {
    return 'description' in item;
  }
```